website_crm_quick_answer/data/automation.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<data noupdate="1">
+
+<record id="website_form_leads_filter" model="ir.filters">
+    <field name="name">Coming from the website contact form</field>
+    <field name="user_id" eval="False"/>
+    <field name="model_id">crm.lead</field>
+    <field name="domain"
+           eval="[('medium_id', '=', ref('utm.utm_medium_website'))]"/>
+</record>
+
+<record id="email_template_quick_response" model="mail.template">
```

Comment:
If I'm not wrong, this record's XMLID was `email_template` before migration. I'd vote for leaving it as it was before, because this is a `noupdate` record, so if anybody changed it, they will probably not like it disappearing after migration, and old name was not so bad.